templates/java/index.html

```diff
@@ -0,0 +1,298 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1sTYQlLM1jWDx-01oC5QY0OdTXNas8KHBoFPR2RW0AQg/edit?tab=t.0#heading=h.6d3m0dei0nxs
+{% endblock meta_copydoc %}
+
+{% block title %}Java support - Canonical builds of OpenJDK for Ubuntu{% endblock %}
+
+{% block meta_description %}
+  Explore Canonical’s Java support offering - long term supported builds of OpenJDK, minimal JRE containers and Spring Boot automation
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row--25-75">
+      <div class="col">
+        {{ image(url="https://assets.ubuntu.com/v1/72583323-open%20JDK%20logo.png",
+                alt="",
+                width="284",
+                height="56",
+                hi_def=True,
+                loading="auto") | safe
+        }}
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h1>Canonical’s build of OpenJDK for Ubuntu</h1>
+          <h2>Trusted, stable and certified builds of OpenJDK with up to 12 years of security maintenance</h2>
```

Comment:
I believe this one should be a `p` with the class `p-heading--2` as the content does not represent a heading but it's stylised as one.

Reply:
Thanks! Done.